packages/common/src/intl/util/IntlUtils.ts

```diff
@@ -0,0 +1,42 @@
+import { i18n } from 'i18next';
```

Comment:
Wasn't sure if name spacing this would be helpful..

`const IntlUtils.useRtl()` 🤷